esp8266-fastled-webserver/Pacifica.cpp

```diff
@@ -0,0 +1,159 @@
+#include "common.h"
```

Comment:
SIMPLE FILE REVIEW

File existed as `Pacifica.h` in many of the active branches.

The main change was to use a common base function for both
the fibonacci and non-fibonacci variants.